raft/src/main/resources/common/message/QuorumStateData.json

```diff
@@ -19,7 +19,6 @@
   "validVersions": "0",
```

Comment:
I guess this change is compatible since the structure is only used in json schema and json is friendly to adding and removing fields, but do you think we should bump this version.

Reply:
Bumping the version is a good point, and add a comment to mention what have changed like [this](https://github.com/apache/kafka/blob/trunk/clients/src/main/resources/common/message/FetchRequest.json#L24) did.

Reply:
Thank you for the feedback. I have addressed this in the latest commit.